src/test/java/com/jcabi/http/request/FakeRequestTest.java

```diff
@@ -68,6 +83,26 @@ public void sendsHttpRequestAndProcessesHttpResponse() throws Exception {
     }
 
     /**
+     * FakeRequest can fetch a fake response.
+     * @throws Exception If something goes wrong inside
+     */
+    @Test
+    public void sendsHttpRequestAndProcessesHttpBinaryResponse()
+        throws Exception {
+        new FakeRequest()
+            .withStatus(HttpURLConnection.HTTP_OK)
+            .withReason(REASON_OK)
+            .withHeader(HttpHeaders.CONTENT_TYPE, MediaType.TEXT_PLAIN)
+            .withBody(BODY_HOW_ARE_YOU.getBytes())
```

Comment:
@antonini if many lines are the same as in other test case maybe we can do some helper class to make it more DRY?